app/src/main/java/ar/com/wolox/android/example/ui/home/HomeActivity.kt

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.example.ui.home
+
+import ar.com.wolox.android.R
+
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+import dagger.Lazy
+import javax.inject.Inject
+
+/**
+ * Home Activity
+ */
+
+class HomeActivity @Inject constructor() : WolmoActivity() {
+
+    @Inject internal lateinit var lazyHomeFragment: Lazy<HomeFragment>
```

Comment:
It's not wrong, but did you make it `Lazy` for some reason or you just copy and paste? 👀

Reply:
It was a copy-paste reaction. After reading some docs I found it's for preventing unnecessary objects initialization. Am I right? (or the docs).